src/openrct2/Cheats.cpp

```diff
@@ -57,6 +57,7 @@ void CheatsReset()
     gameState.Cheats.AllowSpecialColourSchemes = false;
     gameState.Cheats.MakeAllDestructible = false;
     gameState.Cheats.SelectedStaffSpeed = StaffSpeedCheat::None;
+    gameState.Cheats.ForcedParkRating = -1;
```

Comment:
Could you create a constant for this? E.g. `kForcedParkRatingDisabled`.

Reply:
You got it. I changed it just now.

BTW, what does the comment 
TODO: Refactor this. Cheat variables should contain the cheat type and a serialisation method
mean exactly?